router/types/get_quote_request_test.go

```diff
@@ -112,20 +101,15 @@ func TestGetQuoteRequestUnmarshal(t *testing.T) {
 			var result types.GetQuoteRequest
 			err := (&result).UnmarshalHTTPRequest(c)
```

Comment:
Omiting http response, status code check makes  sense, since we are testing `UnmarshalHTTPRequest` that returns an error. But why not check `err` returned from `UnmarshalHTTPRequest` against  `expectedError` as it was previously?

Reply:
Let's track this in a task. Would like to get the bug fix to production